go/runtime/registry/host.go

```diff
@@ -502,11 +504,21 @@ func (n *runtimeHostNotifier) watchKmPolicyUpdates(ctx context.Context, kmRtID *
 
 	n.logger.Debug("watching key manager policy updates", "keymanager", kmRtID)
 
-	// Subscribe to key manager status updates.
+	// Subscribe to key manager status updates (policy might change).
 	stCh, stSub := n.consensus.KeyManager().WatchStatuses()
 	defer stSub.Close()
 
-	// Subscribe to runtime host events.
+	// Subscribe to epoch transitions (quote policy might change).
+	epoCh, sub, err := n.consensus.Beacon().WatchEpochs(ctx)
```

Comment:
@kostko  As key managers have different upgrade procedure as compute runtimes, changing the quote policy on epoch transitions might not be ok as every key manager can run its own version (which might not be the active one, can be higher or lower). I see few options:
- Keep the way it is and expect key manager upgrades to follow versioning.
- Follow versions of key manager nodes and use the latest one.
- Use quote policy of the key manager we are talking to (we would probably need to fetch the quote policy during session initialization, as we cannot know to which key manager the p2p stack will send the request).

Reply:
Hm good point. I think using the active deployment for the quote policy makes the most sense.